source-google-analytics-data-api/tests/snapshots/snapshots__discover__capture.stdout.json

```diff
@@ -16,20 +27,19 @@
           "type": "string"
         },
         "property_id": {
-          "type": "string"
+          "type": [
+            "string"
+          ]
         }
       },
-      "required": [
-        "date",
-        "property_id"
-      ],
       "type": [
+        "null",
         "object"
       ]
     },
     "key": [
-      "/date",
-      "/property_id"
+      "/property_id",
```

Comment:
It's crucially important that we not change around the ordering of keys, and also the schematization of keyed fields like `property_id` here.

There is a good reason why we had previously made them non-nullable (downstream materializations typically cannot support nullable primary keys).

Fixing this will require tweaking the schemas and keys of the connector streams. This is essentially re-applying patches we had previously used in airbyte-to-flow.

(Getting this right is the **central** reason I've been asking for this snapshot -> import -> update snapshots -- so that we can verify in review that nothing is changing unintentionally).

Reply:
Got it, thanks for the info, this made everything more clear, will close the PR